frontend/src/pages/Setup/SetupInkeepChatButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { InkeepChatButtonProps } from '@inkeep/uikit'
+import useInkeepSettings from '@/hooks/useInkeepSettings'
+import { lazy, Suspense } from 'react'
+
+const ChatButton = lazy(() =>
+	import('@inkeep/uikit').then((mod) => ({ default: mod.InkeepChatButton })),
```

Comment:
looks like the chat button isn't visible by default?

https://www.loom.com/share/8a7f2742713d4970bd7c86a202fc6541

Reply:
hmm... seems to be not showing in the reflame preview deployment, but showing fine locally. the custom styles I added are also not being applied to the modal.

Reply:
Just took a look. The custom styles not applying is due to Reflame's lack of support for the public dir (we need everything to be part of your module graph in order to properly cache and invalidate resources).

I believe that can be fixed by moving your css file into /src and then importing it as a url like this:

```
import inkeepCssUrl from '@/inkeep.css'

// ...

stylesheetUrls: [inkeepCssUrl]
```

I'm not quite sure about the `visibility: hidden` thing yet at the moment. Curious if that's also there in your local dev? Maybe something else overrides it that doesn't end up applying in the Reflame preview.

Reply:
@lewisl9029 thanks again - looks like one trick necessary was the `?url` suffix on the import to make sure the value was the URL and not the css file contents - https://vitejs.dev/guide/assets#explicit-url-imports